apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -251,13 +271,6 @@ internal V1.Type ToProtobufType()
         internal static SpannerDbType ForStruct(SpannerStruct spannerStruct) =>
             new SpannerDbType(TypeCode.Struct, spannerStruct.Select(f => new StructField(f.Name, f.Type)).ToList());
 
-        /// <summary>
```

Comment:
While going through this class I also stumbled across this one. It is not in use anywhere and does not really make any sense.

Reply:
+1

(I suspect this is an inherent problem in the "let's introduce a feature latently, but not make it public yet" - when it *does* go public, it can be hard to make sure we catch everything like this.)